cmd/aeraki/main.go

```diff
@@ -70,13 +70,17 @@ func main() {
 		args.ServerID = "Aeraki-" + uuid.New().String()
 	}
 
-	args.PodName = env.RegisterStringVar("POD_NAME", args.ServerID, "").Get()
```

Comment:
Why remove this line?

Reply:
because the log of aeraki are:
```
2023-06-09T09:55:00.259687Z	warn	The environment variable POD_NAME was registered multiple times using different metadata: {POD_NAME   false false 0}, {POD_NAME Aeraki-1d13822c-eca8-470f-a828-57938547f5f2  false false 0} 2023-06-09T09:55:00.259743Z	info	Aeraki parameter: aeraki-xds-address: aeraki.istio-system
```
<img width="812" alt="image" src="https://github.com/aeraki-mesh/aeraki/assets/19339970/a015d94e-a407-43ad-baa4-ae4bd35c6860">
PodName is be registered repeatly.

Reply:
@zhaohuabing